roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1024,6 +1041,7 @@ def generate_facts(self, local_facts, additive_facts_to_overwrite):
         facts = set_flannel_facts_if_unset(facts)
         facts = set_node_schedulability(facts)
         facts = set_master_selectors(facts)
+        facts = set_manageiq_facts_if_unset(facts)
```

Comment:
A method to set use_mangeiq = False if not set seems a bit overkill, instead I would just set use_manageiq=False in get_defaults instead:

```
         common = dict(use_openshift_sdn=True, ip=ip_addr, public_ip=ip_addr,
                       deployment_type='origin', hostname=hostname,
                       public_hostname=hostname, use_manageiq=False)
```